scripts/gon_filepath_editor.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
Instead of deleting this file and hardcoding 4 source paths in 4 gon files, imo we should just make a second script that handles the homebrew gon files to fix the issue that @MuweiHe pointed out.

Something like this:
```
#!/bin/bash

# run sed inside this script to get around difficulties in running it directly inside a post hook
sed "s|path/to/file|$1|" scripts/gon_confluent_$2.hcl > dist/gon_confluent_$2_homebrew.hcl
```

The reason is because our build pipeline already broke once with hardcoded paths after updating goreleaser; their docs say that there is no guarantee that the paths will stay the same after updates.